composer.json

```diff
@@ -27,7 +27,8 @@
     "require-dev": {
         "phpunit/phpunit":           "~5.0",
         "squizlabs/php_codesniffer": "~2.0",
-        "athletic/athletic":         "~0.1.6"
+        "athletic/athletic":         "~0.1.6",
```

Comment:
Athletic and the `GeneratedHydratorPerformance` namespace can be dropped, given the move to phpbench (note that existing tests can be ported)